src/main/java/sx/blah/discord/util/RequestBuffer.java

```diff
@@ -58,7 +56,7 @@ public static void request(IVoidRequest request) {
 	 * @return The number of incomplete requests.
 	 */
 	public static int getIncompleteRequestCount() {
-		return incompleteRequestCount;
+		return requests.size();
```

Comment:
I was so confused why you might cancel the requests if more than one had failed, then I woke up to the remembrance that the keys were mapped to lists.